.template/project/template-contract/tests/functions/core/template.rs

```diff
@@ -0,0 +1,23 @@
+use crate::utils::{interface::core::template, setup::setup};
+
+mod success {
+    use super::*;
+
+    #[tokio::test]
+    async fn pass() {
+        let (instance, _wallet) = setup().await;
+        let _response = template(&instance).await;
+    }
+}
+
+mod revert {
+    use super::*;
+
+    #[tokio::test]
+    #[should_panic]
+    // #[should_panic(expected = "SomeErrorMessage")]
```

Comment:
What are your thoughts on just having the un-commented`#[should_panic(expected = "SomeErrorMessage")]` line instead of `#[should_panic]`?
As this is a template it doesn't need to pass tests and this change would ensure the correct use of `should_panic`.

Reply:
We cannot have this in `CI` if it fails tests and we do not want to let it go out of sync with the version used in the repo. If we want to have a message then we have to add more code into the contract which

1) We have to maintain
2) Someone has to delete each time they copy the template